pyproject.toml

```diff
@@ -0,0 +1,40 @@
+[build-system]
+requires = ["flit"]
+build-backend = "flit.buildapi"
+
+[tool.flit.metadata]
+module = "hypothesis_bio"
+dist-name = "hypothesis-bio"
+author = "Benjamin D. Lee, Luiz Irber"
+author-email = "benjamindlee@me.com, contact@luizirber.org"
+home-page = "https://github.com/Lab41/hypothesis-bio"
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Framework :: Hypothesis",
+    "License :: OSI Approved :: MIT License",
```

Comment:
Shouldn't the license be Apache-2.0?

Reply:
Good catch! https://github.com/Lab41/hypothesis-bio/blob/master/LICENSE says Apache 2.0, but https://github.com/Lab41/hypothesis-bio/blob/a8b041ed6fb2c5febef1859dc9d888144ee7c4e2/setup.py#L101 says MIT. Which one should I follow?

Reply:
Not sure.... @Benjamin-Lee thoughts?

Reply:
It should be Apache-2.0 -- good catch indeed